Index: src/main/java/com/shark/controller/UserController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/com/shark/controller/UserController.java	(date 1553415535000)
+++ src/main/java/com/shark/controller/UserController.java	(date 1553416031815)
@@ -28,6 +28,7 @@
     public String GetUserAll(){
         String all = userService.SelectAll().toString();
         System.out.println(all);
+        
         return all;
     }
 }
